README.md

```diff
@@ -4,28 +4,46 @@
 [![Build Status](https://travis-ci.org/rakutentech/docpub.svg?branch=master)](https://travis-ci.org/rakutentech/docpub)
 [![Coverage Status](https://coveralls.io/repos/github/rakutentech/docpub/badge.svg)](https://coveralls.io/github/rakutentech/docpub)
 
-DocPub is a command line utility for converting a folder structure of markdown files to HTML and uploading the result to Zendesk.
+**DocPub** is a command line utility for converting a folder structure of markdown files to HTML and uploading the result to Zendesk.
 
 ## Getting Started
 
-1. Install the utility globally with `npm install -g docpub`
-2. Set environment variables for your Zendesk username, token, and url by running the following commands:
+### Installing
 
-    - `ZENDESK_API_USERNAME="{Your username}"`
-    - `ZENDESK_API_TOKEN="{Your API token}"`
-    - `ZENDESK_URL="{Your fully qualified Zendesk URL}"`
-
-You can alternative create an enviroment profile in your home directory. Using bash, add the variables to your `.bash_profile`, if there is no file, create it.
+To install the utility, use [npm](https://www.npmjs.org/) `install` command:
 
+```sh
+npm install -g docpub
 ```
-export ZENDESK_API_USERNAME="{Your username}"
-export ZENDESK_API_TOKEN="{Your API token}"
-export ZENDESK_URL="{Your fully qualified Zendesk URL}"
+
+Global installation is used for launching commands.
+
+### Configuring
+
+**DocPub** is configured using a config file. The only supported config file format is `JSON`.
+By default, **DocPub** looks for the config in the root of documentation folder by `docpub.conf` name.
+If needed, specific config location may be set with `--config-path` CLI option. Please note, that
+this option accepts path to the file, not to the folder, where config is located
+
+The minimal config must include following required options:
+
+- `username` - name of ZenDesk user with documentation publish access rights
+- `token` - access token of the user above
```

Comment:
This isn't quite correct: the access token isn't user specific. Once generated, it can be used with any username that has publish rights.

So maybe description should be just `Zendesk API access token`.